src/packages/documents/documents/entity-actions/culture-and-hostnames/modal/culture-and-hostnames-modal.element.ts

```diff
@@ -0,0 +1,249 @@
+			<uui-combobox
+				id="select"
+				label=${this.localize.term('assignDomain_language')}
+				.value=${(this._defaultIsoCode as string) ?? 'inherit'}
+				@change=${this.#onChangeLanguage}>
+				<uui-combobox-list>
+					<uui-combobox-list-option .value=${'inherit'}>
+						${this.localize.term('assignDomain_inherit')}
+					</uui-combobox-list-option>
+					${this.#renderLanguageModelOptions()}
+				</uui-combobox-list>
+			</uui-combobox>
+		</uui-box>`;
+	}
+
+	#renderDomainSection() {
+		return html`<uui-box headline=${this.localize.term('assignDomain_setDomains')}>
+			<umb-localize key="assignDomain_domainHelpWithVariants">
+				Valid domain names are: "example.com", "www.example.com", "example.com:8080", or "https://www.example.com/".<br />Furthermore
+				also one-level paths in domains are supported, eg. "example.com/en" or "/en".
+			</umb-localize>
+			${this.#renderDomains()} ${this.#renderAddNewDomainButton()}
+		</uui-box>`;
+	}
+
+	#renderDomains() {
+		if (!this._domains?.length) return;
+		return html`<div id="domains">
+			${repeat(
+				this._domains,
+				(domain) => domain.isoCode,
+				(domain, index) => html`
+					<uui-input
+						label=${this.localize.term('assignDomain_domain')}
+						.value=${domain.domainName}
+						@change=${(e: UUIInputEvent) => this.#onChangeDomainHostname(e, index)}></uui-input>
+					<uui-combobox
+						.value=${domain.isoCode as string}
+						label=${this.localize.term('assignDomain_language')}
+						@change=${(e: UUISelectEvent) => this.#onChangeDomainLanguage(e, index)}>
+						<uui-combobox-list> ${this.#renderLanguageModelOptions()} </uui-combobox-list>
+					</uui-combobox>
+					<uui-button
+						look="outline"
+						color="danger"
+						label=${this.localize.term('assignDomain_remove')}
+						@click=${() => this.#onRemoveDomain(index)}>
+						<uui-icon name="icon-trash"></uui-icon>
+					</uui-button>
+				`,
+			)}
+		</div>`;
+	}
+
+	#renderLanguageModelOptions() {
+		return html`${repeat(
+			this._languageModel,
+			(model) => model.isoCode,
+			(model) => html`<uui-combobox-list-option .value=${model.isoCode}>${model.name}</uui-combobox-list-option>`,
+		)}`;
+	}
+
+	#renderAddNewDomainButton() {
+		return html`<uui-button-group>
+			<uui-button
+				label=${this.localize.term('assignDomain_addNew')}
+				look="placeholder"
+				@click=${() => this.#onAddDomain()}></uui-button>
+			<uui-button
+				id="dropdown"
+				label=${this.localize.term('buttons_select')}
+				look="placeholder"
+				popovertarget="more-options">
+				<uui-icon name="icon-navigation-down"></uui-icon>
+			</uui-button>
+			<uui-popover-container id="more-options" placement="bottom-end">
+				<umb-popover-layout>
+					<uui-button label=${this.localize.term('assignDomain_addCurrent')} @click=${() => this.#onAddDomain(true)}>
+						<umb-localize key="assignDomain_addCurrent"> Add current domain </umb-localize>
+					</uui-button>
```

Comment:
I dont think you need to localize this two times over as long as you leave out any whitespace in the button:

```suggestion
					<uui-button label=${this.localize.term('assignDomain_addCurrent')} @click=${() => this.#onAddDomain(true)}></uui-button>
```

Reply:
Yeah, probably added the localizations a little too fast without thinking about it.